### PR TITLE
[zh-cn] Sync docs/reference/using-api

### PR DESCRIPTION
**Pull request by tico88612**
- `content/en/docs/reference/using-api/api-concepts.md`
- `content/en/docs/reference/using-api/cel.md`
- `content/en/docs/reference/using-api/deprecation-guide.md`
- `content/en/docs/reference/using-api/health-checks.md`
- `content/en/docs/reference/using-api/server-side-apply.md`

### CODE REVIEW

File: content/zh-cn/docs/reference/using-api/deprecation-guide.md [hunk 1]
```diff
@@ -157,13 +156,17 @@ The **autoscaling/v2beta2** API version of HorizontalPodAutoscaler is no longer
...
+  * `targetAverageUtilization` is replaced with `target.averageUtilization` and `target.type: Utilization`. See [Autoscaling on multiple metrics and custom metrics](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics).
 -->
 从 v1.26 版本开始不再提供 **autoscaling/v2beta2** API 版本的
 HorizontalPodAutoscaler。
 
 * 迁移清单和 API 客户端使用 **autoscaling/v2** API 版本，
   此 API 从 v1.23 版本开始可用；
 * 所有的已保存的对象都可以通过新的 API 来访问。
+* 值得注意的变更：
+  * `targetAverageUtilization` 被替换为 `target.averageUtilization` 和 `target.type: Utilization`。请参阅[基于多项度量指标和自定义度量指标自动扩缩](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics)。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/using-api/deprecation-guide.md:169]: We can replace this with `zh-cn` version.

```suggestion
  * `targetAverageUtilization` 被替换为 `target.averageUtilization` 和 `target.type: Utilization`。请参阅[基于多项度量指标和自定义度量指标自动扩缩](/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics)。
```

File: content/zh-cn/docs/reference/using-api/deprecation-guide.md [hunk 2]
```diff
@@ -263,12 +266,16 @@ The **autoscaling/v2beta1** API version of HorizontalPodAutoscaler is no longer
...
+* Notable changes:
+  * `targetAverageUtilization` is replaced with `target.averageUtilization` and `target.type: Utilization`. See [Autoscaling on multiple metrics and custom metrics](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics).
 -->
 从 v1.25 版本开始不再提供 **autoscaling/v2beta1** API 版本的
 HorizontalPodAutoscaler。
 
 * 迁移清单和 API 客户端使用 **autoscaling/v2** API 版本，此 API 从 v1.23 版本开始可用；
 * 所有的已保存的对象都可以通过新的 API 来访问。
+* 值得注意的变更：
+  * `targetAverageUtilization` 被替换为 `target.averageUtilization` 和 `target.type: Utilization`。请参阅[基于多项度量指标和自定义度量指标自动扩缩](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics)。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/using-api/deprecation-guide.md:278]:
```suggestion
  * `targetAverageUtilization` 被替换为 `target.averageUtilization` 和 `target.type: Utilization`。
  请参阅[基于多项度量指标和自定义度量指标自动扩缩](/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics)。
```

File: content/zh-cn/docs/reference/using-api/health-checks.md [hunk 3]
```diff
@@ -131,12 +131,12 @@ healthz check passed
...
 <!-- 
 Each individual health check exposes an HTTP endpoint and can be checked individually.
-The schema for the individual health checks is `/livez/<healthcheck-name>` where `livez` and `readyz` and be used to indicate if you want to check the liveness or the readiness of the API server.
+The schema for the individual health checks is `/livez/<healthcheck-name>` or `/readyz/<healthcheck-name>`, where `livez` and `readyz` can be used to indicate if you want to check the liveness or the readiness of the API server, respectively.
 The `<healthcheck-name>` path can be discovered using the `verbose` flag from above and take the path between `[+]` and `ok`.
 These individual health checks should not be consumed by machines but can be helpful for a human operator to debug a system:
 -->
 每个单独的健康检查都会公开一个 HTTP 端点，并且可以单独检查。
-单个运行状况检查的模式为 `/livez/<healthcheck-name>`，其中 `livez` 和 `readyz` 表明你要检查的是 API 服务器是否存活或就绪。
+单个运行状况检查的模式为 `/livez/<healthcheck-name>` 或 `/readyz/<healthcheck-name>`，其中 `livez` 和 `readyz` 分别表明你要检查的是 API 服务器是否存活或就绪。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/using-api/health-checks.md:139]:
```suggestion
单个运行状况检查的模式为 `/livez/<healthcheck-name>` 或 `/readyz/<healthcheck-name>`，
其中 `livez` 和 `readyz` 分别表明你要检查的是 API 服务器是否存活或就绪。
```

File: content/zh-cn/docs/reference/using-api/server-side-apply.md [hunk 4]
```diff
@@ -326,12 +326,16 @@ as [YAML](https://yaml.org/), with the media type `application/apply-patch+yaml`
...
+parser that does not fully implement the YAML specification. Some JSON escapes may
+not be recognized.
 -->
 不管你提交的是 JSON 数据还是 YAML 数据，
 都要使用 `application/apply-patch+yaml` 作为 `Content-Type` 的值。
 
-所有的 JSON 文档 都是合法的 YAML。
+所有的 JSON 文档都是合法的 YAML。不过，Kubernetes 存在一个缺陷，
+即它使用的 YAML 解析器没有完全实现 YAML 规范。
+某些 JSON 转义可能无法识别。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/using-api/server-side-apply.md:338]:
```suggestion
某些 JSON 转义可能无法被识别。
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**